Library/Homebrew/formula.rb

```diff
@@ -2920,16 +2920,16 @@ def service(&block)
     # the {Formula} will be built from source and `reason` will be printed.
```

Comment:
Yeah, the fact that I was confused for a second as to why this `reason` wasn't changed is a good indicator that this wording is better 👍